hoaxy/commands/site.py

```diff
@@ -796,6 +808,57 @@ def run(cls, args):
                 logger.warning('Site %s does not exist!', site_identity)
             else:
                 cls.enable_site(session, msite)
+        # bulk enable sites and domains
+        elif args['--bulk-enable'] is True:
+            configure_logging(
+                'site.bulk-enable',
+                console_level=args['--console-log-level'],
+                file_level='WARNING')
+            if args['--exclusive'] is True:
+                ob_expr = Site.id.asc()
+                msites = get_msites(session, fb_kw=None, ob_expr=ob_expr)
+                # disable existing sites
+                for existing_site in msites:
+                    cls.disable_site(session, existing_site)
+            if args['--names'] is not None:
+                site_list = args['--names']
+                for site in site_list:
+                    msite = qquery_msite(session, name=site, domain=None)
+                    if msite is None:
+                        logger.warning('Site %s does not exist!', site)
+                    else:
+                        cls.enable_site(session, msite)
+            else:
+                domain_list = args['--domains']
+                for domain in domain_list:
+                    msite = qquery_msite(session, name=None, domain=domain)
+                    if msite is None:
+                        logger.warning('Site %s does not exist!', domain)
+                    else:
+                        cls.enable_site(session, msite)
+
+        # bulk disable sites and domains
```

Comment:
For `bulk-disable` operation (starting line 840), `split` action is not needed because the argument is parsed by `docopt`. The logical seems correct for me. Did you do the test? You can test the command to see whether the changes of the corresponding table are as expected.

Good Luck!